### DISCUSSION TITLE
Is qt6-multimedia required, when compiling audacious-plugins-git ?

### DISCUSSION THREAD

**Discussion by ffs-akl** [audacious-media-player/audacious#1415] [Q&A]
When I compile audacious-plugins 4.4, the compiler outputs a warning:
Run-time dependency qt6 (modules: Multimedia) found: YES 6.7.1 (pkg-config)
audacious-plugins/src/qtaudio/meson.build:7: WARNING: The Qt Multimedia output plugin is not yet compatible with Qt 6.

I have installed the qt6-multimedia,  qt6-multimedia-ffmpeg,qt6-multimedia-gstreamer.

**Comment by mschwendt** [comment]
It's not the compiler that outputs that warning, but the build system files.
See https://github.com/audacious-media-player/audacious-plugins/blob/master/src/qtaudio/meson.build

The warning itself is clear and concise, isn't it? If building with Qt 6, you cannot build the qtaudio output plugin for Audacious. Do you need that one? Most people output to ALSA or PulseAudio these days.

**Comment by ffs-akl** [comment]
I don't need qtaudio output.  thank you, I'm just worried if it's going to have an effect on the compiled program.
If it's just one of the audio outputs, it won't be a problem.

>The warning itself is clear and concise, isn't it?
This warning could easily be misinterpreted.

**Comment by ffs-akl** [comment]
I don't need qtaudio output.  thank you, I'm just worried if it's going to have an effect on the compiled program.
If it's just one of the audio outputs, it won't be a problem.

>The warning itself is clear and concise, isn't it?


This warning could easily be misinterpreted.
